src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageQueueNameConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageQueueNameConverter
+    {
+        public static string GetEndpointUrl(string accountName, string queueName)
+        {
```

Comment:
Should we return a Uri object instead of a string?

Reply:
So i followed the pattern in MsmqQueueNameConverter. Over there it is returning a string as well. 
I can see why it should return an URI object though, so if you think thats the right way , i can update this,